MiaServiceDotNetLibrary/Logging/LoggingExtensions.cs

```diff
@@ -5,9 +5,10 @@ namespace MiaServiceDotNetLibrary.Logging
     public static class LoggingExtensions
     {
         public static IApplicationBuilder UseRequestResponseLoggingMiddleware(
-            this IApplicationBuilder builder)
+            this IApplicationBuilder builder, MiddlewareOptions options = default)
```

Comment:
what is default here?

Also: this brings a breaking change to the application using this method.

I'd provide an overloaded version that allows the user to specify custom excluded prefix.

In the method without the options parameter I'd create a `new MiddlewwareOptions` that by default excludes status routes prefix

Reply:
*default* means that it will take all the properties that I have defined as default in `MiddlewareOptions` class if no instance of `MiddlewareOptions` gets passed as a parameter to `UseRequestResponseLoggingMiddleware`. In this case, right now, the default value for `excludedPrefixes` property is an empty list. What I think is best to do is set as default value for the property `excludedPrefixes` a list with only one string inside: `"/-/"`. In this way, if no `MiddlewareOptions` instance gets passed as parameter the status routes will not show any log. In this way, there will be no breaking change and, if someone wants to exclude more routes, they only need to pass a `MiddlewareOptions` instance with the list of excluded prefixes (including also the `"/-/"` to avoid logging status routes).

Reply:
I have modified the default value for `excludedPrefixes` property as I said! Now this change should not be breaking anymore! 
`UseRequestResponseLoggingMiddleware` function can now be called both without parameters, which means that it will filter status routes from logging, or with a `MiddlewareOptions` parameter that specifies which prefixes to exclude, like in the following way:
`UseRequestResponseLoggingMiddleware(new MiddlewareOptions {excludedPrefixes = new List<string>() { "/-/", "/anotherRouteToExclude/" }});`
It is also necessary to add the following line in the `using` section to be able to use `List`:
`using System.Collections.Generic;`

Reply:
ok, let's keep the default if it's not breaking.

Let's apply `/-/` as default value anyway so that application using the library does not have to know where status routes are exposed.